### ISSUE TITLE
Validator warns that Markdown autolinks will be escaped

### ISSUE THREAD

**Issue by pieter-edelman-nictiz** [hapifhir/org.hl7.fhir.validator-wrapper#133]
Hi,

We have several profiles where an `ElementDefinition.comment` contains [Markdown autolinks](https://github.github.com/gfm/#autolinks), e.g.:

```
Either `.policy` or `.policyRule` should exist following core constraint ppc-1. If no explicit value is recorded, a fallback is to populate `.policy.uri` with &lt;https://wetten.overheid.nl/&gt; if the advance directive is recorded following Dutch law and regulations.
```

(so it has an absolute URL formatted in between `<` and `>`).

When validating such a profile, the Validator gives the following warning:

> The markdown contains content that appears to be an embedded HTML tag starting at 'Ei'. This will (or SHOULD) be escaped by the presentation layer. The content should be checked to confirm that this is the desired behaviour

I believe this warning is incorrect. The syntax is valid according to GFM for specifying an URL and a valid GFM parser should be able to distinguish it from an embedded HTML tag.

**Comment by grahamegrieve** [comment]
I don't see how it's incorrect. It appears to be an embedded html tag. As you say, it's valid, and processed properly will be ok. The warning is just pointing out that it needs to be processed correctly

**Comment by pieter-edelman-nictiz** [comment]
Issue has been resolved :)
